catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/CatalogInputAdapter.java

```diff
@@ -47,26 +50,32 @@ public ResourceRequest buildReadRequest(Metacard metacard, String qualifier) {
       throw new IllegalArgumentException("method argument 'qualifier' may not be null.");
     }
 
-    Attribute attribute = metacard.getAttribute(Metacard.DERIVED_RESOURCE_URI);
+    Attribute attribute = metacard.getAttribute(Core.RESOURCE_URI);
 
     if (attribute != null) {
-      List<Serializable> values = attribute.getValues();
+      Serializable value = attribute.getValue();
 
-      if (values == null) {
+      if (value == null) {
         throw new IllegalStateException(
-            String.format(
-                "metacard attribute %s has no values assigned.", Metacard.DERIVED_RESOURCE_URI));
+            String.format("metacard attribute %s has no value assigned.", Core.RESOURCE_URI));
       }
 
-      URI qualifiedUri = findDerivedResourceUri(values, qualifier);
+      Map<String, Serializable> props = new HashMap<>();
+      props.put(ContentItem.QUALIFIER, qualifier);
 
-      return new ResourceRequestByProductUri(qualifiedUri);
+      try {
+        return new ResourceRequestByProductUri(new URI((String) value), props);
+      } catch (URISyntaxException e) {
+        throw new IllegalArgumentException(
+            String.format(
+                "The supplied metacard does contains an invalid '%s' attribute.",
```

Comment:
`The supplied metacard contains an invalid '%s' attribute.`

Reply:
@peterhuffer did you forget to update the exception message here?

Reply:
Yeah, looks like it 😩I might have ended up messing it up again too if it was different before :)